### PR TITLE
chore: update assets

### PR DESCRIPTION
**Pull request by SeMiD**
*beep boop* I've updated some assets...

### CODE REVIEW

File: websites/Z/ZenMarket/metadata.json [hunk 1]
```diff
@@ -13,8 +13,8 @@
 		"*://zenmarket.jp/*"
 	],
 	"version": "1.0.4",
-	"logo": "https://i.ibb.co/YWQTYzh/cy0lojzy.png",
-	"thumbnail": "https://zenmarket.jp/img/about/about_top_bg.jpg",
+	"logo": "https://cdn.rcd.gg/PreMiD/websites/Z/ZenMarket/assets/logo.png",
```
Comment (theusaf) [inline websites/Z/ZenMarket/metadata.json:16]: Source image appears to no longer exist

File: websites/G/Geode/metadata.json [hunk 2]
```diff
@@ -17,8 +17,8 @@
 		"*://docs.geode-sdk.org/*"
 	],
 	"version": "1.0.4",
-	"logo": "https://i.joerkig.com/gowo8n.png",
-	"thumbnail": "https://i.joerkig.com/s99rcj.png",
+	"logo": "https://cdn.rcd.gg/PreMiD/websites/G/Geode/assets/logo.png",
+	"thumbnail": "https://cdn.rcd.gg/PreMiD/websites/G/Geode/assets/thumbnail.png",
```
Comment (theusaf) [inline websites/G/Geode/metadata.json:21]: Source image server was offline

File: websites/C/ChompuBot/metadata.json [hunk 3]
```diff
@@ -14,8 +14,8 @@
 		"*://chompubot.work/*"
 	],
 	"version": "1.0.2",
-	"logo": "https://chompubot.work/logo.jpg",
-	"thumbnail": "https://chompubot.work/thumbnail.jpg",
+	"logo": "https://cdn.rcd.gg/PreMiD/websites/C/ChompuBot/assets/logo.jpg",
+	"thumbnail": "https://cdn.rcd.gg/PreMiD/websites/C/ChompuBot/assets/thumbnail.jpg",
```
Comment (theusaf) [inline websites/C/ChompuBot/metadata.json:18]: Thumbnail failed to upload for some reason. Image too big?

File: websites/B/BeatLeader/presence.ts [hunk 4]
```diff
@@ -16,11 +16,11 @@ const presence = new Presence({
...
-		beatleader: "https://i.joerkig.com/27hnf5.png",
-		nomods: "https://i.joerkig.com/6ob2zh.png",
-		nopause: "https://i.joerkig.com/qg0376.png",
-		golf: "https://i.joerkig.com/2ow21y.png",
-		scpm: "https://i.joerkig.com/5lewbj.png",
+		beatleader: "https://cdn.rcd.gg/PreMiD/websites/B/BeatLeader/assets/114.png",
+		nomods: "https://cdn.rcd.gg/PreMiD/websites/B/BeatLeader/assets/115.png",
+		nopause: "https://cdn.rcd.gg/PreMiD/websites/B/BeatLeader/assets/116.png",
+		golf: "https://cdn.rcd.gg/PreMiD/websites/B/BeatLeader/assets/117.png",
+		scpm: "https://cdn.rcd.gg/PreMiD/websites/B/BeatLeader/assets/118.png",
```
Comment (theusaf) [inline websites/B/BeatLeader/presence.ts:23]: Source image server was offline

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/Z/ZenMarket/metadata.json (hunk 1 above)
Comment (Bas950) [inline websites/Z/ZenMarket/metadata.json:16]:
```suggestion
	"logo": "https://i.ibb.co/Sc5jnK9/image.png",
```

File: websites/Z/ZenMarket/presence.ts [hunk 5]
```diff
@@ -4,7 +4,7 @@ const presence = new Presence({
 	browsingTimestamp = Math.floor(Date.now() / 1000);
 
 const enum Assets {
-	Logo = "https://i.ibb.co/YWQTYzh/cy0lojzy.png",
+	Logo = "https://cdn.rcd.gg/PreMiD/websites/Z/ZenMarket/assets/logo.png",
```
Comment (Bas950) [inline websites/Z/ZenMarket/presence.ts:7]:
```suggestion
	Logo = "https://i.ibb.co/Sc5jnK9/image.png",
```

File: websites/C/ChompuBot/metadata.json (hunk 3 above)
Comment (Bas950) [inline websites/C/ChompuBot/metadata.json:18]:
```suggestion
	"thumbnail": "https://i.ibb.co/YPt4Htv/thumbnail-1.jpg",
```

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by theusaf — APPROVED** [review]

**Pull request merged**